packages/ui5-middleware-livetranspile/README.md

```diff
@@ -65,6 +69,10 @@ The transpiled code and the `sourcemap` are subsequently delivered to the client
 
 > `async/await` is transpiled at runtime, but the required `asyncGenerator` sources are not yet delivered on the fly. They need to be `sap.ui.require`d or `<script src="...">`d separately. Alternatively you can use the babel plugin `babel-plugin-transform-async-to-promises` as described [here](../ui5-task-transpile/README.md).
 
+## Extended configuration (in `$yourapp/babel.config.json`)
```

Comment:
again just a wording thing: I'd suggest "extending the default configuration..." instead of "Extended configuration..."